### ISSUE TITLE
The role of corporations and Open Street Map

### ISSUE THREAD

**Issue by paulmach** [mvexel/ama#2]
On the one hand, corporate interests generally don't match well to the altruistic goals of OSM. On the other, corporations have a lot of resources (money and otherwise) that could greatly benefit the project. How do you see role of corporations (from Telenav, Mapbox, Strava... all the way to Apple) in the open street map community?

**Comment by mvexel** [comment]
Thanks Paul for the question. My own role at Telenav exemplifies how I see this relation. it is a `oneway=false`. 

On the one hand, I take my almost 10 years of OSM experience - both mapping and working with OSM communities on different scales - and try to inject that knowledge into the organization. Not just the technical stuff but also the values of the community. So that key people within Telenav really do understand OSM. This helps us as a company make better decisions for ourselves **and** the community. An example is the way Scout's transition to OSM turned out. We actively sought input from the US community on topics like [freeway exit tagging](http://www.openstreetmap.org/user/mvexel/diary/22419). I believe the discussions that ensued helped both us make a better product and the OSM community converge around conventions that made sense to all of us. After Scout's initial release, we [added the capability](http://www.openstreetmap.org/user/mvexel/diary/28208) to add OSM notes from within the app. A non-intrusive way to get feedback from Scout users, most of whom don't even know what OSM is.

![screenshot 2015-11-11 13 56 54](https://cloud.githubusercontent.com/assets/120452/11492206/ffe3509c-97a6-11e5-9164-9708c89459df.png)

On the other hand, I look at what we have to offer as a company and how we can share that with the OSM community. This has lead to [a](http://www.openstreetmap.org/user/mvexel/diary/21270) [number](http://www.openstreetmap.org/user/mvexel/diary/23975) [of](http://www.openstreetmap.org/user/mvexel/diary/22389) MapRoulette challenges (and [my work](http://www.openstreetmap.org/user/mvexel/diary/21192) [on MapRoulette](http://www.openstreetmap.org/user/mvexel/diary/23374) itself) and more recently custom tools that identify [errors](http://www.openstreetmap.org/user/mvexel/diary/36209) and [missing roads](http://www.openstreetmap.org/user/mvexel/diary/35966) in the OSM data based on our massive store of Scout user GPS data. My mission is to keep coming up with new ways to share what we can.

![screenshot 2015-11-23 15 29 34](https://cloud.githubusercontent.com/assets/120452/11492333/db311bf6-97a8-11e5-82ba-71ff97c6ae31.png)

So to summarize - I think corporate interests **can actually align really well** with OSM's mission, but it requires a mutual desire to understand each other. Having someone who, like me, is deeply invested in OSM be part of your organization I think is essential to achieve success.
